about.html

```diff
@@ -0,0 +1,191 @@
+          <h2 class="second-about-heading">
+            Humanoid robots, Human-robot interaction. <br />
+              and Distributed Control.
+          </h2>
+          <p class="second-about-paragraph">
+            Humanoid robots, Human-robot interaction and Distributed Control was decided through
+            the logo competition from 27, June to 9, July.
+          </p>
+          <img
+            src="./images/logo 2.jpg"
+            alt="logo"
+            class="img-logo"
+          />
+        </div>
+    </section>
+       <!-- last conference held on -->
+      <section id="third-section">
+        <div class="section-third">
+          <h2 class="second-about-heading">
+            See the past Humanoid robots
+            and Distributed Control summits
+          </h2>
+          <p>
+            Take a look at the last two Humanoid robots and Distributed 
+            Control Summits which took place in Bahir Dar and Adiss Ababa.
+          </p>
+
+          <div class="continues-programs">
+            <a href="#" class="about-program" target="_blank" rel="noopener noreferrer">
+              <img src="./images/hawasa.jpg" alt="images Addisababa" class="img-about"/>
+              <span class="cover"></span>
+              <h3>2020</h3>
+              <p>Bahirdar</p>
+            </a>
+            <a href="#" class="about-program" target="_blank" rel="noopener noreferrer">
+              <img
+                src="./images/Addisababa.jpg"
+                alt="images Addisababa"
+              />
+              <span class="cover"></span>
+              <h3>2022</h3>
+              <p>Addis Ababa</p>
+            </a>
+          </div>
+        </div>
+      </section>
+      <!-- Sponsors(partners)- section -->
+    <section class="about-sponsors">
+        <h2 class="sponsers-heading">Partner</h2>
+        <div class="sponser-underline"></div>
+        <div class="sponsors-organization">
+          <a href="https://www.google.com/">
+            <img src="./images/sponsor-1.png" title="Google" alt="Google"
+          /></a>
+          <a href="https://www.nasa.gov/">
+            <img src="./images/sponsor-2.png" title="Nasa" alt="Nasa"
+          /></a>
+          <a href="https://www.spacex.com/">
+            <img src="./images/sponsor-3.png" title="SpaceX" alt="SpaceX"
+          /></a>
+          <a href="https://www.roscosmos.ru/">
+            <img
+              src="./images/sponsor-4.png"
+              title="Roscosmos"
+              alt="Roscosmos"
+          /></a>
+          <a href="https://www.airbnb.com/">
+            <img src="./images/sponsor-5.png" title="airbnb" alt="airbnb" />
+          </a>
+        </div>
+      </section>
+  
+      <!-- Footer -->
+      <section class="footer">
+        <div class="foot"></div>
+        <div class="footer-main">
+          <div class="footer-image">
+            <img src="./images/logo 2.jpg" alt="footer-image" />
+          </div>
```

Comment:
- [ **OPTIONAL** ] What do you think if you use your logo as a link so it can be used to be redirected to the home section to make your project more ptresional.